src/cider/nrepl/middleware/info.clj

```diff
@@ -87,7 +87,7 @@
                 (when (= "file" (.getProtocol rsrc))
                   (io/as-file rsrc))
                 (io/as-file filename))]
-    (when (and file (.exists file))
+    (when (and file (.exists file) (.isFile file))
```

Comment:
@expez a `File` can represent either a file or a directory, and if it's a directory, the following call to `io/input-stream` will throw.

@pleasetrythisathome just to clarify, you have a directory on the classpath whose name is something like `core.cljs`, and `find-cljx-source` is trying to open that instead of the actual `core.cljs`?